src/main/java/core/basesyntax/service/operations/PurchaseOperationsHandler.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.model.Operations;
+
+public class PurchaseOperationsHandler implements OperationsHandler {
```

Comment:
According to the checklist, there should not be an empty line at the beginning of the class implementation.